wikitexthtml/page.py

```diff
@@ -30,8 +31,10 @@ def __init__(self, page: str) -> None:
         self._errors = []  # type: List[str]
         self._template_path = []  # type: List[str]
 
-    def prepare(self, body) -> wikitextparser.WikiText:
-        body = preprocess.begin(self, body)
+    def prepare(self, body, should_color_syntax=False) -> wikitextparser.WikiText:
+        body = preprocess.begin(self, body, should_color_syntax=should_color_syntax)
+        if should_color_syntax:
+            body = preprocess.code_block(self, body)
```

Comment:
`code_block` feels a bit weird. `syntax_highlight`?

Reply:
How about `highlight_syntax`?

Reply:
Haha, so now everything is "syntax highlight" expect for this :P My OCD thinks a thing or two about that :P But no really, it is fine :) What-ever floats your boat :) At least it is lovely verbose, so I like it :)